tubes/lua.lua

```diff
@@ -229,11 +229,13 @@ end
 -------------------------
 
 local function safe_print(param)
-	local string_meta = getmetatable("")
-	local sandbox = string_meta.__index
-	string_meta.__index = string -- Leave string sandbox temporarily
-	print(dump(param))
-	string_meta.__index = sandbox -- Restore string sandbox
+	if minetest.settings:get("pipeworks_lua_tube_print_behavior") == "log" then
```

Comment:
Moving in default value from global settings table
```suggestion
	if (minetest.settings:get("pipeworks_lua_tube_print_behavior") or "log") == "log" then
```